plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/DockerInstanceProvider.java

```diff
@@ -144,7 +144,13 @@ public DockerInstanceProvider(DockerConnector docker,
         commonEnvVariables = new String[0];
         devMachineEnvVariables = new String[] {API_ENDPOINT_URL_VARIABLE + '=' + apiEndpoint,
                                                DockerInstanceMetadata.PROJECTS_ROOT_VARIABLE + '=' + PROJECTS_FOLDER_PATH};
-        this.machineExtraHosts = isNullOrEmpty(machineExtraHosts) ? null : machineExtraHosts.split(",");
+        String[] extraHosts = isNullOrEmpty(machineExtraHosts) ? null : machineExtraHosts.split(",");
+        String dockerHost = "che-host:".concat(docker.getDockerHostIp());
+        if (extraHosts == null) {
```

Comment:
We don't need ternary operator 2 lines before because you added this if statement. Please change to
```
String dockerHost = "che-host:".concat(docker.getDockerHostIp());
if (isNullOrEmpty(machineExtraHosts)) {
    this.machineExtraHosts = new String[] {dockerHost};
} else {
    this.machineExtraHosts = ObjectArrays.concat(machineExtraHosts.split(","), dockerHost);
}
```